---
title: "RSAT::Plants: Motif Discovery in ChIP-Seq Peaks of Plant Genomes"
collection: publications
permalink: /publication/2016-08-25_RSAT-plants-1_paper
excerpt: '...'
date: 2016-08-25
venue: 'Methods in Molecular Biology'
paperurl: 'https://doi.org/10.1007/978-1-4939-6396-6_19'
citation: 'Castro-Mondragon J.A., Rioualen C., Contreras-Moreira B., van Helden J. (2016) RSAT::Plants: Motif Discovery in ChIP-Seq Peaks of Plant Genomes. In: Hehl R. (eds) Plant Synthetic Promoters. Methods in Molecular Biology, vol 1482. Humana Press, New York, NY'
---

**Abstract**

In this protocol, we explain how to run ab initio motif discovery in order to gather putative transcription factor binding motifs (TFBMs) 
from sets of genomic regions returned by ChIP-seq experiments. The protocol starts from a set of peak coordinates (genomic regions) 
which can be either downloaded from ChIP-seq databases, or produced by a peak-calling software tool. We provide a concise description 
of the successive steps to discover motifs, cluster the motifs returned by different motif discovery algorithms, and compare them with 
reference motif databases. The protocol is documented with detailed notes explaining the rationale underlying the choice of options. 
The interpretation of the results is illustrated with an example from the model plant Arabidopsis thaliana.



