---
title: "7th Santorini Conference on Biology Prospective"
collection: talks
type: "Poster"
permalink: /talks/2014-09-24_Santorini_poster
venue: "Petros M. Nomikos Conference Centre"
date: 2014-09-24
location: "Fira, Santorini, Greece"
---

[European Society on Pharmacogenomics and Personalized Therapy](https://esptnet.eu)

Interactome–regulome–transcriptome integrative approach as a mean to disclose cancer stem cells regulatory circuits 

C Rioualen, R El-Helou, E Charafe-Jauffret, C Ginestier, G Bidaut

Centre de Recherche en Cancérologie de Marseille, Inserm U1068, CNRS UMR7258, Aix-Marseille Université, Institut Paoli-Calmettes, Marseille, 13009, France.

Background. Several attempts have been made to build signatures for the prognosis of breast cancer (BC). However, these signatures showed a low reproducibility. Thus the idea of using high-throughput data integration rose, and more stable signatures were obtained1.
Objective. We decided to use this approach in order to identify bi-modal genes regulated by miR-600 in breast cancer stem cells (CSC).
Design. After a functional whole genome screen of an miRNA library, miR-600 was identified as a modulator of CSC self-renewal and differentiation. We measured gene expression in CSCs after miR-600 overexpression and knock-down. 
We then applied a network-based integrative approach using a heavily modified version of the ITI algorithm1. ITI has been initially developed in order to perform gene-expression-based tumor classification, using cancer cells expression and a map of publicly available protein-protein interactions data.
In order to isolate pathways involved in CSC evolution, we added a regulation map (regulome made of TF-target couples), using interactions from Tred and Transfac. 
In this new algorithm, the interactome and the regulome are separately investigated, using target genes as “seeds”. If the seed's score is above a given threshold it is put aside. Its neighboring nodes are aggregated recursively if they improve the subnetwork's score, calculated by averaging screening scores of genes it contains. It is then statistically validated by comparing its score to those of subnetworks generated with randomized data. Interactome and regulome subnetworks are then merged by overlap. Resulting “metasubnetworks” are believed to be regulation pathways involved in CSC survival or death.
Results. This two-step integration revealed interesting modules, including genes known to regulate Wnt signaling pathways. A Gene Ontology enrichment search further suggested that miR-600 could be a switch in the regulation of CSCs.
Conclusions. Integrating gene expression data and network information allows the identification of biological pathways involved in CSC biology. These findings constitute a basis for anti-CSC drug target discovery in BC.
Acknowledgements: Support to this project is coming from the Institut National du Cancer grant Number INCA_5911 to CG.
Reference.
1Garcia M, Millat-Carus R, Bertucci F, Finetti P, Birnbaum D, Bidaut G. Interactome-transcriptome integration for predicting distant metastasis in breast cancer. Bioinformatics. 2012 Mar 1;28(5):672-8.


Poster available [here](http://rioualen.github.io/files/2014-09-24_Santorini_poster.pdf)


