---
title: "7th Santorini Conference on Biology Prospective"
collection: talks
type: "Talk"
permalink: /talks/2014-09-24_Santorini_talk
venue: "Petros M. Nomikos Conference Centre"
date: 2014-09-24
location: "Fira, Santorini, Greece"
---

[European Society on Pharmacogenomics and Personalized Therapy](https://esptnet.eu)

Cancer stem cells regulatory circuit is deciphered through an interactome–regulome–transcriptome integrative approach.

C Rioualen, R El-Helou, E Charafe-Jauffret, C Ginestier, G Bidaut

Centre de Recherche en Cancérologie de Marseille, Inserm U10681, CNRS UMR72582, Aix-Marseille Université3, Institut Paoli-Calmettes4, Marseille, 13009, France.

Background: Tumor-initiating cells have been characterized in multiple cancers, including breast cancer (BC). So-called cancer stem cells (CSCs) present key stem cell properties including self-renewal (which drives tumorigenesis) and differentiation. They show resistance to conventional cancer therapies, and are thought to be the seed for the distant metastasis responsible for poor clinical outcomes. Unraveling deregulated mechanisms that are sustaining self-renewal and differentiation specific to CSCs is an essential step for the discovery of new research avenues and treatment in BC.
Objective: We aim to identify regulatory circuits driving breast CSC biology using a systems biology approach. The discovery of new druggable targets specific to the CSC population would allow refining personalized treatments in BC. These modules include information on regulation (transcription factors) as well as post-regulation (miRNA).
Design: After a functional whole genome screen of a miRNA library, we identified miR-600 as a modulator of CSC self-renewal and differentiation. We measured gene expression in CSCs after miR-600 overexpression and knock-down. We then used a two-step integrated approach based on the Interactome-Transcriptome Integration (ITI) algorithmref in order to identify genetic modules post-regulated by miR-600. First, we identified modules with significantly deregulated expression in the human interactome (protein-protein interactions). Second, we identified pathways regulated by miR-600 in CSCs, using ITI and a regulation map based on TRANSFAC data (Protein-DNA interactions).
Results: By crossing the results of this two-step integrated approach, we established a list of deregulated modules in CSCs. We found 10 miR-600-regulated network modules driving CSC differentiation. Several transcription factors (RBM5, CEBPA, NMUR2), tumors suppressors (EGR1, TP53), and genes known to regulate stem cell pathways (WNK1, a regulator of Wnt signaling) were identified.
Conclusions: Integrating gene expression data and network information (including physical interactions and regulatory relationships) allows the identification of biological pathways involved in CSC biology. These findings constitute a basis for anti-CSC drug target discovery in BC.
Acknowledgements: Support to this project is coming from the Institut National du Cancer grant Number INCA_5911 to CG.

Reference
Garcia M, Millat-Carus R, Bertucci F, Finetti P, Birnbaum D, Bidaut G. Interactome-transcriptome integration for predicting distant metastasis in breast cancer. Bioinformatics. 2012 Mar 1;28(5):672-8.


Slides available [here](http://rioualen.github.io/files/2014-09-24_Santorini_slides.pdf)


