---
title: "17th Open Days in Biology, Computer Science and Mathematics (JOBIM) "
collection: talks
type: "Poster"
permalink: /talks/2016-06-28_JOBIM_poster
venue: "École normale supérieure de Lyon"
date: 2016-06-28
location: "Lyon, France"
---

[Official website off the event](https://jobim2016.sciencesconf.org/)
[Proceedings](https://jobim2016.sciencesconf.org/conference/jobim2016/pages/book_jobim2016.pdf)

Snakemake workflows deployed on virtual environments: a promising way of integrating high-throughput data in RegulonDB

Claire Rioualen, Lucie Khamvongsa, Alberto Santos-Zavaleta, Mishael Sánchez-Pérez, Jocelyn Brayet, Julio Collado-Vides, Jacques van Helden

Technologies avancées pour le génome et la clinique (TAGC), INSERM U1090 , Aix-Marseille Université, Marseille F-13288, France [website](https://tagc.univ-amu.fr/)
Computational Genomics Research program, Center for Genomics Sciences - UNAM, México [website](http://www.ccg.unam.mx/)
Institut Curie, PSL Research University, Mines Paris Tech, Inserm, U900, F-75005, Paris, France.

*Introduction* Next-generation sequencing (NGS) has become a mainstream technology in genomics, and it
has gotten increasingly cheaper and faster to obtain genomic data. However, the development
of reliable tools for the analysis of the huge amount of data generated is still lagging behind.
We have developed workflows using the Snakemake environment, for its scalability and ease of
implementation. Then we proceeded with building a catalogue of tools for ChIP-seq analyses, that
were implemented in custom workflows. This work was integrated in different virtual environments,
and used to re-analyse published data as well as manually curated data from RegulonDB.

[RegulonDB database](http://regulondb.ccg.unam.mx)

Poster available [here](http://rioualen.github.io/files/2016-06-28_JOBIM_poster.pdf)


